app/workers/authorize_all_follows_worker.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class AuthorizeAllFollowsWorker
+  include Sidekiq::Worker
+
+  def perform(account_id)
+    account = Account.find(account_id)
+    FollowRequest.where(target_account: account).each  do |req|
+      AuthorizeFollowWorker.perform_async(req.account_id, req.target_account_id)
```

Comment:
Having one query instead of lots of `Account.find` in `AuthorizeFollowWorker` would be better. The latency is not really important for this operation, and the overall load of workers would take precedence.
```Ruby
account = Account.find(account_id)
FollowRequest.where(target_account: account).includes(:account)
```
…where `.includes(:account)` is optional. I think it is fine to drop it and leave for future optimization when necessary.